src/styles/typo.css.ts

```diff
@@ -1,4 +1,139 @@
-export const typo = {
-  fontSize: {},
-  fontWeight: {},
+import { createGlobalTheme, style } from '@vanilla-extract/css';
+
+export const fontVars = createGlobalTheme(':root', {
```

Comment:
이 fontVars를 `createGlobalTheme()` 사용 없이 일반 객체로 선언하는 건 어떤가요? `createGlobalTheme()`은 다크모드 구현 시에는 유용할 것 같은데, 다크모드 구현 계획이 없는 지금으로서는 일반 객체로 선언해도 차이가 없을 것 같아서 불필요하다면 빼도 좋을 것 같다는 의견입니다 ㅎㅎ
```suggestion
export const fontVars = {
```

Reply:
한영님 말씀대로 지금은 단일테마만 적용 중이니 나중에 다크모드가 추가되면 `createGlobalTheme()`을 사용하는 것이 더 좋을 것 같습니다! 수정해서 다시 올리겠습니다 :)